frame/aura/src/lib.rs

```diff
@@ -86,7 +89,9 @@ decl_storage! {
 }
 
 decl_module! {
-	pub struct Module<T: Trait> for enum Call where origin: T::Origin { }
+	pub struct Module<T: Trait> for enum Call where origin: T::Origin {
+		const EpochDuration: EraIndex = T::Period::get();
```

Comment:
I believe the "epoch" wording is babe-specific, would just call it "SessionDuration" (or "SessionLength", whatever makes sense in the context it is used in)